packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -60,17 +60,21 @@ const getColorVariant = (hashValue) => {
 /**
  * Render placeholder.
  */
-const generateImagePlaceholder = (alt, isAriaHidden, variant) => {
-  const avatarIconClassNames = cx(['icon', variant]);
+const generateImagePlaceholder = (alt, isAriaHidden, variant, initials) => {
```

Comment:
want to update this to accept an object containing these properties? something i learned recently :)
```suggestion
const generateImagePlaceholder = ( 
  { alt, isAriaHidden, variant, initials }
) => {
```

Reply:
Do I need to create object containing these properties [here](https://github.com/cerner/terra-core/blob/9b6a5d4ec8ec4ced52e8cacbcb62be86f2a35395/packages/terra-avatar/src/variants/Avatar.jsx#L87) like below and to pass that object to generateImagePlaceholder() function..? 

```javascript
const avatarAttributes = {
   alt : 
   isAriaHidden :
   variant : 
   initials : 
}
```

Reply:
yup!

Reply:
changed to object params : [here](https://github.com/cerner/terra-core/blob/565f1417aaeb24a8bcc372b80c65ba24ebe360ab/packages/terra-avatar/src/variants/Avatar.jsx#L88-L101)